modules/vector-search/pages/create-vector-search-index-ui.adoc

```diff
@@ -0,0 +1,74 @@
+= Create a Vector Search Index in Quick Mode
+:page-topic-type: guide
+:description: Use Quick Mode to create a Vector Search index in Couchbase Capella.
+
+[abstract]
+{description}
+
+Use xref:create-search-index-ui.adoc[Advanced Mode] to add additional settings and more advanced options for your Vector Search index and xref:customize-index.adoc#type-mappings[type mappings].
```

Comment:
Broken links (I think it's supposed to be in the `:search:` directory?)

Reply:
Thought I caught all these from my copy/paste - thanks!